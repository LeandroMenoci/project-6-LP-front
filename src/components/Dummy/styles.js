/*eslint-disable */
import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css``}
`;
