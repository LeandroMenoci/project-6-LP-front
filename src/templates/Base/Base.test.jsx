import { Base } from '.';
import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';

describe('<Base/>', () => {
  it('Should render', () => {
    renderTheme(<Base>Children</Base>);
    expect(screen.getByRole('heading')).toBeInTheDocument();
  });
});
