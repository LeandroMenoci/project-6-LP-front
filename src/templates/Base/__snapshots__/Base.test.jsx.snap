// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base/> Should render 1`] = `
<div>
  <button
    aria-label="Open/Close menu"
    class="sc-gKXOVf dGLJZS"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="StyledIconBase-ea9ulj-0 lbJwfL"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    class="sc-eCYdqJ hKylYu"
  >
    <div
      class="sc-dkzDqf hjcBda"
    >
      <div
        class="sc-jSMfEi lflkom"
      >
        <h1
          class="sc-hKMtZM gPfEAk"
        >
          <a
            class="sc-iBkjds jaaffS"
          >
            <span>
              Logo
            </span>
          </a>
        </h1>
        <nav
          aria-label="Main menu"
          class="sc-ftvSup elltSo"
        >
          <a
            class="sc-papXJ bhUkag"
            href="#target1"
            target="_self"
          >
            Link 1
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target2"
            target="_self"
          >
            Link 2
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target3"
            target="_self"
          >
            Link 3
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target4"
            target="_self"
          >
            Link 4
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target5"
            target="_self"
          >
            Link 5
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target6"
            target="_self"
          >
            Link 6
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target7"
            target="_self"
          >
            Link 7
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target8"
            target="_self"
          >
            Link 8
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target9"
            target="_self"
          >
            Link 9
          </a>
          <a
            class="sc-papXJ bhUkag"
            href="#target10"
            target="_self"
          >
            Link 10
          </a>
        </nav>
      </div>
    </div>
  </div>
  <div
    class="sc-gsnTZi ixfVzt"
  >
    <div
      class="sc-breuTD jEFKeo"
    >
      <div
        class="sc-dkzDqf hjcBda"
      >
        <div
          class="sc-iqcoie dTFQIy"
        >
          <h2
            class="sc-hKMtZM lgLECr"
          >
            My Grid
          </h2>
          <div
            class="sc-bczRLJ ijhGli"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam fugiat provident, qui sequi est ad, eos
          </div>
          <div
            class="sc-crXcEl lmcECn"
          >
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 1
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 2
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 3
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-breuTD ksagHO"
    >
      <div
        class="sc-dkzDqf hjcBda"
      >
        <div
          class="sc-iqcoie dTFQIy"
        >
          <h2
            class="sc-hKMtZM hjixfg"
          >
            My Grid
          </h2>
          <div
            class="sc-bczRLJ ijhGli"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam fugiat provident, qui sequi est ad, eos
          </div>
          <div
            class="sc-crXcEl lmcECn"
          >
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 1
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 2
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 3
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-breuTD jEFKeo"
    >
      <div
        class="sc-dkzDqf hjcBda"
      >
        <div
          class="sc-iqcoie dTFQIy"
        >
          <h2
            class="sc-hKMtZM lgLECr"
          >
            My Grid
          </h2>
          <div
            class="sc-bczRLJ ijhGli"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam fugiat provident, qui sequi est ad, eos
          </div>
          <div
            class="sc-crXcEl lmcECn"
          >
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 1
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 2
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 3
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-breuTD ksagHO"
    >
      <div
        class="sc-dkzDqf hjcBda"
      >
        <div
          class="sc-iqcoie dTFQIy"
        >
          <h2
            class="sc-hKMtZM hjixfg"
          >
            My Grid
          </h2>
          <div
            class="sc-bczRLJ ijhGli"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam fugiat provident, qui sequi est ad, eos
          </div>
          <div
            class="sc-crXcEl lmcECn"
          >
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 1
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 2
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 3
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-breuTD jEFKeo"
    >
      <div
        class="sc-dkzDqf hjcBda"
      >
        <div
          class="sc-iqcoie dTFQIy"
        >
          <h2
            class="sc-hKMtZM lgLECr"
          >
            My Grid
          </h2>
          <div
            class="sc-bczRLJ ijhGli"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam fugiat provident, qui sequi est ad, eos
          </div>
          <div
            class="sc-crXcEl lmcECn"
          >
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 1
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 2
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM exvzzo"
              >
                Teste 3
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-breuTD ksagHO"
    >
      <div
        class="sc-dkzDqf hjcBda"
      >
        <div
          class="sc-iqcoie dTFQIy"
        >
          <h2
            class="sc-hKMtZM hjixfg"
          >
            My Grid
          </h2>
          <div
            class="sc-bczRLJ ijhGli"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam fugiat provident, qui sequi est ad, eos
          </div>
          <div
            class="sc-crXcEl lmcECn"
          >
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 1
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 2
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
            <div
              class="sc-evZas cfTEap"
            >
              <h3
                class="sc-hKMtZM fgIiWP"
              >
                Teste 3
              </h3>
              <div
                class="sc-bczRLJ ijhGli"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-jqUVSM cTayaa"
    >
      <div
        class="sc-dkzDqf hjcBda"
      >
        <div
          class="sc-bczRLJ ijhGli"
        >
          <p>
            Teste de footer
          </p>
        </div>
      </div>
    </div>
  </div>
  <a
    aria-label="Go to top"
    class="sc-kDDrLX hqgpMh"
    href="#"
  >
    <svg
      aria-hidden="true"
      class="StyledIconBase-ea9ulj-0 lbJwfL"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
